ovirt-imageio.spec.in

```diff
@@ -13,17 +13,41 @@ Source0:   https://github.com/oVirt/%{name}/releases/download/v%{version}/%{name
 %global admin_confdir %{_sysconfdir}/%{name}
 %global vendor_confdir %{_prefix}/lib/%{name}
 
+# We need to have Python 3.8 support of imageio client for oVirt Ansible
+# Collection running with ansible-core-2.12, which uses non-standard Python 3.8
+# on EL8 platform
+%if "0%{?rhel}" == "08"
```

Comment:
Shouldn't the check be `%if 0%{?rhel} > 7 && 0%{?rhel} < 9`, or `%if "0%{?rhel}" == 8`?

Reply:
I hate `%if 0%{?rhel} > 7 && 0%{?rhel} < 9` and `%if 0%{?rhel} == 8` never worked for me, so trying a new variant, let's see :-)

Reply:
Interesting I had no problem running `%if 0%{?rhel} == 8` inside my el8 container.